Assets/Treasure/TDK/Runtime/API/Magicswap.cs

```diff
@@ -140,7 +141,29 @@ public class NFTInput {
         public List<NFTInput> nftsIn = null;
         public List<NFTInput> nftsOut = null;
         public double? slippage = null;
-        public string backendWallet = null;
+        public string backendWallet = null; // TODO should this be set to TDKConfig.GetBackendWallet()?
+    }
+
+    [Serializable]
+    public class AddLiquidityBody {
+        public List<NFTInput> nfts0 = null;
+        public List<NFTInput> nfts1 = null;
+        public string amount0;
+        public string amount1;
+        public string amount0Min;
+        public string amount1Min;
+        public string backendWallet = null; // TODO should this be set to TDKConfig.GetBackendWallet()?
+    }
+
+    [Serializable]
+    public class RemoveLiquidityBody {
+        public List<NFTInput> nfts0 = null;
+        public List<NFTInput> nfts1 = null;
+        public string amountLP;
+        public string amount0Min;
+        public string amount1Min;
+        public bool? swapLeftover = null;
```

Comment:
Please default this to `true`

Reply:
This is defaulted to true [in the api](https://github.com/TreasureProject/tdk-js/blob/main/apps/api/src/schema/magicswap.ts#L211), i was just omitting it here with `null` so it takes that default, but i can set it explicitly if you think thats better 👀

Reply:
Done!